### PR TITLE
Unit test solution

### PR DESCRIPTION
**Pull request by meaghanlewis**

### CODE REVIEW

File: UnitTestProject2/UnitTestProject2/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("UnitTestProject2")]
```
Comment (ghfvs-tester) [inline UnitTestProject2/UnitTestProject2/Properties/AssemblyInfo.cs]: nice

File: UnitTestProject2/UnitTestProject2/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
+[assembly: Guid("c2aacbbf-d54a-4f72-a8b4-b14807a1c069")]
+
+// Version information for an assembly consists of the following four values:
+//
+//      Major Version
+//      Minor Version 
+//      Build Number
+//      Revision
```
Comment (ghfvs-tester) [inline UnitTestProject2/UnitTestProject2/Properties/AssemblyInfo.cs]: nice

**Review by ghfvs-tester — APPROVED** [review]

**Review by ghfvs-tester — APPROVED** [review]

File: UnitTestProject2/UnitTestProject2/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+[assembly: AssemblyTitle("UnitTestProject2")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("UnitTestProject2")]
+[assembly: AssemblyCopyright("Copyright ©  2018")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+// Setting ComVisible to false makes the types in this assembly not visible
```
Comment (ghfvs-tester) [inline UnitTestProject2/UnitTestProject2/Properties/AssemblyInfo.cs]: cool

**Review by ghfvs-tester — APPROVED** [review]

File: UnitTestProject2/UnitTestProject2/UnitTest1.cs [hunk 4]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UnitTestProject2
+{
+    [TestClass]
```
Comment (meaghanlewis) [inline UnitTestProject2/UnitTestProject2/UnitTest1.cs]: LGTM

**Review by meaghanlewis — COMMENTED** [review]
"My review"

**Review by ghfvs-tester — CHANGES_REQUESTED** [review]
looking good, but a few comments to address

**Pull request closed without merging**